### DISCUSSION TITLE
How to combine operator in a where condition ?

### DISCUSSION THREAD

**Discussion by sandwoodK** [nushell/nushell#6690] [General]
I can´t get this simple condition working. Any help appreciated. 

`ps | where status == Sleeping and (name =~ "bash" or name =~ "nu")`

**Comment by fdncred** [comment]
maybe something like this would work `ps | where status == Sleeping | where name =~ "bash" or name =~ "nu"`

**Reply by sandwoodK** [reply]
Thanks. This works. Any idea why the above does not work ? it looks like a bug to me.

**Reply by fdncred** [reply]
no idea. maybe it would work kind of like this works `ls | where {|i| ($i.name starts-with p) and ($i.name =~ package or $i.name =~ parser) }`
